src/main/java/appeng/util/Platform.java

```diff
@@ -656,6 +669,33 @@ else if( o instanceof ItemStack )
 		}
 	}
 
+	public static String getFluidDisplayName( Object o )
+	{
+		if( o == null )
+		{
+			return "** Null";
+		}
+		FluidStack fluidStack = null;
+		if( o instanceof AEFluidStack )
+		{
+			fluidStack = ( (AEFluidStack) o ).getFluidStack();
+		}
+		else if( o instanceof FluidStack )
+		{
+			fluidStack = (FluidStack) o;
+		}
+		else
+		{
+			return "**Invalid Object";
+		}
+		String n = fluidStack.getLocalizedName();
+		if( n == null || n.equalsIgnoreCase( "" ) )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1132)